plugin/src/main/java/hudson/plugins/swarm/PluginImpl.java

```diff
@@ -70,6 +92,12 @@ public void doCreateSlave(StaplerRequest req, StaplerResponse rsp, @QueryParamet
                 }
                 jenkins.addNode(slave);
             }
+            rsp.setContentType("text/plain; UTF-8");
+            byte[] response = name.getBytes("UTF-8");
+            rsp.setContentLength(response.length);
```

Comment:
A fair point

On Friday, April 24, 2015, James Nord <notifications@github.com> wrote:

> In plugin/src/main/java/hudson/plugins/swarm/PluginImpl.java
> <https://github.com/jenkinsci/swarm-plugin/pull/26#discussion_r29064189>:
>
> > @@ -70,6 +92,12 @@ public void doCreateSlave(StaplerRequest req, StaplerResponse rsp, @QueryParamet
> >                  }
> >                  jenkins.addNode(slave);
> >              }
> > +            rsp.setContentType("text/plain; UTF-8");
> > +            byte[] response = name.getBytes("UTF-8");
> > +            rsp.setContentLength(response.length);
>
> may I recommend adding a terminating \r\n (or \n) or perhaps even a
> swarmName=blah rather than just blah so that it is easier to evolve this
> in the future if need be without breaking older clients.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jenkinsci/swarm-plugin/pull/26/files#r29064189>.
>


-- 
Sent from my phone